src/index.ts

```diff
@@ -51,6 +51,11 @@ export interface ServerlessClamscanLoggingProps {
   readonly logsPrefix?: string;
 }
 
+export interface BucketEventNotificationProps {
+  readonly bucketName?: string;
```

Comment:
**suggestion:** I don't think it makes sense for `bucketName` to be optional.

```suggestion
  readonly bucketName: string;
```

Reply:
I wanted to make this field mandatory but it seems that for someone who wants to use addSourceBucket() he has to populate bucketName even though it is already provided in bucket (IBucket type).
At the other end, bucketName is necessary because I have map it to the corresponding bucket in buckets list.